atlite/convert.py

```diff
@@ -539,23 +539,34 @@ def wind(cutout, turbine, smooth=False, add_cutout_windspeed=False, **params):
 
 # irradiation
 def convert_irradiation(
-    ds, orientation, irradiation="total", trigon_model="simple", clearsky_model=None
+    ds,
+    orientation,
+    tracking,
```

Comment:
Maybe `tracking=None` in order not to introduce a new compulsory argument?

Reply:
Good point. Done.